x/leverage/keeper/math_test.go

```diff
@@ -1,10 +1,12 @@
-package keeper
+package keeper_test
```

Comment:
These are unit tests, so no need to change the test package type.

Reply:
Noted, I'll revert it

Reply:
thanks